doc/source/getting_started/using_julia.rst

```diff
@@ -0,0 +1,192 @@
+.. _using_julia:
+
+
+***********************
+Using PyMAPDL in Julia
+***********************
+
+If you like to work with Julia, you can use python libraries as they were Julia packages.
+
+
+Installing Julia
+=================
+
+To install Julia go to their website `<https://julialang.org/>`_ and follow the instructions given in the *Download* section.
+
+* `Windows <https://julialang.org/downloads/platform/#windows>`_
+* `Linux <https://julialang.org/downloads/platform/#linux_and_freebsd>`_
+* `MacOS <https://julialang.org/downloads/platform/#macos>`_
+
+Setting Julia Environment
+==========================
+
+To have access to Python libraries, we need to install *PyCall* Julia package.
+To install it, run Julia and switch to the package manager by pressing ``"]"`` key.
+
+It is beneficial to create a virtual environment in Julia, in order to work with different package versions or applications if needed.
+To create a virtual environment, use ``activate`` command with the name of the new environment you want to create or activate.
+
+.. code-block:: julia
+
+    (@1.7) pkg> activate julia_test
+      Activating project at `C:\Users\USER\julia_test`
+
+
+It should show a message that the new package (``julia_test``) has been activated and that environment name will now precede the command line.
+
+.. code-block:: julia
+
+    (julia_test) pkg>
+
+Then we proceed to install *PyCall* package by typing:
+
+.. code-block:: julia
+
+    (julia_test) pkg> add PyCall
+
+
+To use *PyCall*, we go first to the Julia command line by pressing the backspace key.
+Then we can use the *PyCall* package as:
+
+.. code-block:: julia
+
+    julia> using PyCall
+
+
+This should be enough to use packages included in a basic Python distribution. 
+
+
+For example:
+
+.. code-block:: julia
+
+    julia> math = pyimport("math")
+    math.sin(math.pi/4) # returns ≈ 1/√2 = 0.70710678..
+
+
+Installing PyMAPDL in Julia
+============================
+
+*PyCall* includes a lightweight Python environment which uses `Conda <https://conda.io>`_ to manage and access Python packages.
+This environment is based on Python 3.9.7 (at the moment of writing this article) and includes the standard basic Python libraries.
+However since it is a fully working Python environment, we can still use it from outside Julia command line and install Python packages using *pip*.
```

Comment:
```suggestion
However, because it is a fully working Python environment, you can still use it from outside the Julia command line and install Python packages using ``pip``.
```